modules/swagger-codegen/src/main/resources/python/api_client.mustache

```diff
@@ -165,10 +163,9 @@ class ApiClient(object):
         :param obj: The data to serialize.
         :return: The serialized form of data.
         """
-        types = (str, float, bool, bytes) + tuple(integer_types) + (text_type,)
-        if isinstance(obj, type(None)):
+        if obj is None:
             return None
-        elif isinstance(obj, types):
+        elif isinstance(obj, (float, bool, bytes, text_type) + integer_types):
```

Comment:
@frol What about storing `(float, bool, bytes, text_type) + integer_types)` in a class static variable so as to avoid the tiny overhead in concatenation?